daemon/mgr/container.go

```diff
@@ -829,6 +830,16 @@ func (mgr *ContainerManager) parseVolumes(ctx context.Context, c *types.Containe
 			}
 
 			source = mountPath
+		} else {
```

Comment:
`func parseVolumes` is not so good, and there are so much improvement we could do for it.